AndroidAnnotations/androidannotations-core/androidannotations-test/src/test/java/org/androidannotations/test/efragment/MyListFragmentTest.java

```diff
@@ -123,6 +123,28 @@ public void ignoredWhenDetachedWorksForIgnoredMethod() {
 	}
 
 	@Test
+	public void ignoredWhenViewDestroyedForIgnoredMethod() {
+		assertFalse(myListFragment.didExecute);
+		myListFragment.onDestroyView();
+		myListFragment.ignoreWhenViewDestroyed();
+		assertFalse(myListFragment.didExecute);
+	}
+
+	@Test
+	public void notIgnoredBeforeDetached() {
+		assertFalse(myListFragment.didExecute);
+		myListFragment.ignored();
+		assertTrue(myListFragment.didExecute);
+	}
+
+	@Test
+	public void notIgnoredBeforeViewDestroyed() {
+		assertFalse(myListFragment.didExecute);
+		myListFragment.ignoreWhenViewDestroyed();
+		assertTrue(myListFragment.didExecute);
+	}
+
```

Comment:
Sorry, but i meant another thing:
destroyView
createView
callMethod
assert didExecute true

This is the case when a `Fragment` is returned from the back stack.
The current test is not that interesting, because basically it is the same as `ignoredWhenViewDestroyedForIgnoredMethod`.

You can just fix it without creating a separate commit.